lua/no-neck-pain/config.lua

```diff
@@ -17,52 +17,62 @@ NoNeckPain.options = {
     -- When `true`, disabling NNP kills every split/vsplit buffers except the main NNP buffer.
     killAllBuffersOnDisable = false,
     -- Options related to the side buffers.
+    -- When `true`, the side buffers will be named `no-neck-pain-left` and `no-neck-pain-right` respectively.
+    showBufferNames = false,
     buffers = {
-        -- The background options of the side buffer(s).
-        background = {
-            -- Hexadecimal color code to override the current background color of the buffer. (e.g. #24273A)
-            -- popular theme are supported by their name:
-            -- - catppuccin-frappe
-            -- - catppuccin-latte
-            -- - catppuccin-macchiato
-            -- - catppuccin-mocha
-            -- - tokyonight-day
-            -- - tokyonight-moon
-            -- - tokyonight-night
-            -- - tokyonight-storm
-            -- - rose-pine
-            -- - rose-pine-moon
-            -- - rose-pine-dawn
-            colorCode = nil,
-        },
-        -- When `false`, the `left` padding buffer won't be created.
-        left = true,
-        -- When `false`, the `right` padding buffer won't be created.
-        right = true,
-        -- When `true`, the side buffers will be named `no-neck-pain-left` and `no-neck-pain-right` respectively.
-        showName = false,
-        -- The buffer options when creating the buffer.
-        options = {
-            -- Buffer-scoped options, below are the default values, but any `vim.bo` options are valid and will be forwarded to the buffer creation.
+        left = {
+            enabled = true,
```

Comment:
```suggestion
            -- When `false`, the buffer won't be created.
            enabled = true,
```

Same for the `right` one